lib/Conversation/CommentForm/index.js

```diff
@@ -28,31 +29,43 @@ class CommentForm extends Component {
 
   constructor() {
     super();
-    this.state = { inputValue: '' };
+    this.state = { inputValue: '', focused: false, submitted: false };
     this.updateInputValue = this.updateInputValue.bind(this);
     this.cancelComment = this.cancelComment.bind(this);
     this.onSubmit = this.onSubmit.bind(this);
+    this.onFocus = this.onFocus.bind(this);
+    this.onBlur = this.onBlur.bind(this);
   }
 
   onSubmit(e) {
     e.preventDefault();
     this.props.onSubmit(this.state.inputValue);
-    this.setState({ inputValue: '' });
+    this.props.existingConversation && (
+      this.setState({ inputValue: '' })
+    );
+    this.setState({ submitted: true });
```

Comment:
Does React combined these 2 setStates together? If not then we're calling setState twice rather than calling it once and maybe causing 2 renders for 1 UI update, is that an issue?

Something like this would combine the 2 👍 

```
const inputValue = this.props.existingConversation ? '' : this.state.inputValue
this.setState({ 
  submitted: true,
  inputValue,
})
```